### ISSUE TITLE
Runtime Tests flake in 08-nat46

### ISSUE THREAD

**Issue by aalemayhu** [cilium/cilium#1363]
See attachment for more or https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-1288/41/execution/node/28/log/

```
00:21:12.894 ==> cilium-master-build-PR-1288-41: Aug 24 05:15:25 cilium-master cilium-agent[4006]: time="2017-08-24T05:15:25-07:00" level=debug msg="Processing an event {Status:exec_start: ping6 -c 5 ::FFFF:10.11.83.26 ID:3eea2c86113e0f7e1988615b1e92c10808319cd837d21e5ca5cd22fa0b08043c From:tgraf/netperf Type:container Action:exec_start: ping6 -c 5 ::FFFF:10.11.83.26 Actor:{ID:3eea2c86113e0f7e1988615b1e92c10808319cd837d21e5ca5cd22fa0b08043c Attributes:map[id.client: image:tgraf/netperf name:client]} Time:1503576925 TimeNano:1503576925644438848}"
00:21:12.894 ==> cilium-master-build-PR-1288-41: Aug 24 05:15:29 cilium-master cilium-agent[4006]: time="2017-08-24T05:15:29-07:00" level=info msg="Regenerated program of endpoint 62502"
00:21:12.894 ==> cilium-master-build-PR-1288-41: Aug 24 05:15:30 cilium-master cilium-agent[4006]: time="2017-08-24T05:15:30-07:00" level=info msg="Regenerated program of endpoint 49896"
00:21:12.894 ==> cilium-master-build-PR-1288-41: 
00:21:12.894 ==> cilium-master-build-PR-1288-41: ------------------------------------------------------------------------
00:21:12.894 ==> cilium-master-build-PR-1288-41: ./helpers.bash: line 56: 17609 Terminated              cilium monitor -v $@ > $DUMP_FILE
00:21:12.894 ==> cilium-master-build-PR-1288-41: cleanup
00:21:12.894 ==> cilium-master-build-PR-1288-41: + TEST_NAME=08-nat46
```

[cilium-files-runtime-PR-1288-41.tar.gz](https://github.com/cilium/cilium/files/1248972/cilium-files-runtime-PR-1288-41.tar.gz)
[cilium-files-k8s-PR-1288-41.tar.gz](https://github.com/cilium/cilium/files/1248973/cilium-files-k8s-PR-1288-41.tar.gz)
[short.output.txt](https://github.com/cilium/cilium/files/1248974/short.output.txt)

**Comment by aalemayhu** [comment]
```
[Runtime Tests] ==> cilium-master-build-PR-1288-41: ------------------------------------------------------------------------
[Runtime Tests] ==> cilium-master-build-PR-1288-41:                             Test Failed
[Runtime Tests] ==> cilium-master-build-PR-1288-41: Error: Could not ping nat64 address of client from host
[Runtime Tests] ==> cilium-master-build-PR-1288-41: 
[Runtime Tests] ==> cilium-master-build-PR-1288-41: ------------------------------------------------------------------------
[Runtime Tests] ==> cilium-master-build-PR-1288-41: + abort 'Error: Could not ping nat64 address of client from host'
[Runtime Tests] ==> cilium-master-build-PR-1288-41: + set +x
```

**Comment by tgraf** [comment]
Fixed by #1398
